Multisig/Cross-layer/FirebaseRemoteConfig.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  FirebaseRemoteConfig.swift
+//  Multisig
+//
+//  Created by Moaaz on 4/15/21.
+//  Copyright © 2021 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+import Firebase
+
+class FirebaseRemoteConfig {
+    static let shared = FirebaseRemoteConfig()
+    enum Key: String {
+        case newestVersion
+        case deprecatedSoon
+        case deprecated
+    }
+
+    private var remoteConfig: RemoteConfig!
+    private let defaultValues: [String : NSObject] = [Key.newestVersion.rawValue : App.configuration.app.marketingVersion as NSObject,
```

Comment:
this is not correct - the default newest version value must be null or empty

Reply:
I think it's fine to have value for it like this, but I change it anyway